docs/tutorials/1_config.md

```diff
@@ -398,231 +434,230 @@ which is convenient to conduct various experiments.
+        test_dataloader=dict(   # Additional config of testing dataloader
+            videos_per_gpu=1),   # Batch size of each single GPU during testing
+        train=dict(   # Training dataset config
+            type=dataset_type,
+            ann_file=ann_file_train,
+            exclude_file=exclude_file_train,
+            pipeline=train_pipeline,
+            label_file=label_file,
+            proposal_file=proposal_file_train,
+            person_det_score_thr=0.9,
+            data_prefix=data_root),
+        val=dict(     # Validation dataset config
+            type=dataset_type,
+            ann_file=ann_file_val,
+            exclude_file=exclude_file_val,
+            pipeline=val_pipeline,
+            label_file=label_file,
+            proposal_file=proposal_file_val,
+            person_det_score_thr=0.9,
+            data_prefix=data_root))
+    data['test'] = data['val']    # Set test_dataset as val_dataset
+    # optimizer
+    optimizer = dict(
+        # Config used to build optimizer, support (1). All the optimizers in PyTorch
+        # whose arguments are also the same as those in PyTorch. (2). Custom optimizers
+        # which are builed on `constructor`, referring to "tutorials/5_new_modules.md"
+        # for implementation.
+        type='SGD',  # Type of optimizer, refer to https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/optimizer/default_constructor.py#L13 for more details
+        lr=0.2,  # Learning rate, see detail usages of the parameters in the documentaion of PyTorch (for 8gpu)
+        momentum=0.9,  # Momentum,
+        weight_decay=0.00001)  # Weight decay of SGD
+
+    optimizer_config = dict(  # Config used to build the optimizer hook
+        grad_clip=dict(max_norm=40, norm_type=2))   # Use gradient clip
+
+    lr_config = dict(  # Learning rate scheduler config used to register LrUpdater hook
+        policy='step',  # Policy of scheduler, also support CosineAnnealing, Cyclic, etc. Refer to details of supported LrUpdater from https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/lr_updater.py#L9
+        step=[40, 80],  # Steps to decay the learning rate
+        warmup='linear',  # Warmup strategy
+        warmup_by_epoch=True,  # Warmup_iters indicates iter num or epoch num
+        warmup_iters=5,   # Number of iters or epochs for warmup
+        warmup_ratio=0.1)   # The initial learning rate is warmup_ratio * lr
+
+    total_epochs = 20  # Total epochs to train the model
+    checkpoint_config = dict(  # Config to set the checkpoint hook, Refer to https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/checkpoint.py for implementation
+        interval=1)   # Interval to save checkpoint
+    workflow = [('train', 1)]   # Workflow for runner. [('train', 1)] means there is only one workflow and the workflow named 'train' is executed once
+    evaluation = dict(  # Config of evaluation during training
+        interval=1)  # Interval to perform evaluation
```

Comment:
`evaluation = dict(interval=1, save_best='mAP@0.5IOU')`

I think it's better to put `evaluation` on the next line of `data`. Logically, `evaluation` is a dataset setting.